### PR TITLE
[device & platform] led and transceiver deamon for inventec switch d7032 and d7054

### PR DESCRIPTION
**Pull request by klhaung**
**- What I did**
* Add front panel led support for Inventec-d7032
* Add transceiver monitor daemon for Inventec-d7054
* Follow new porting guide to have modified broadcom config under device folder
* BugFix in sfputil.py used for inventec-d7032

**- How I did it**
* Add two pythons scripts which is spawned by debian init script for monitoring port status
* Add led initialization file for Inventec-d7032
* Have modified broadcom config under device folder for Inventec-d7032 and Inventec-d7054

**- How to verify it**
* Build image, install on device platform to and make sure it is worked properly

**- Description for the changelog**

Add front panel led support and transceiver monitor for invnetec d7054 and d7032

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-inventec/common/utils/transceiver_monitor.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Inventec, Inc.
+# 
+# Editor: James Huang ( Huang.James@inventec.com )
+#  
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-inventec/common/utils/transceiver_monitor.py]: can you remove license from the file.

File: platform/broadcom/sonic-platform-modules-inventec/common/utils/led_proc.py [hunk 2]
```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Inventec, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-inventec/common/utils/led_proc.py]: can you remove license from the file.

Comment (klhaung) [reply platform/broadcom/sonic-platform-modules-inventec/common/utils/led_proc.py]: sure. done in commit 4ddf83d84a84f583843948c01fc581c6ebcb0c71

File: platform/broadcom/sonic-platform-modules-inventec/common/utils/transceiver_monitor.py (hunk 1 above)
Comment (klhaung) [reply platform/broadcom/sonic-platform-modules-inventec/common/utils/transceiver_monitor.py]: sure, done in commit commit 4ddf83d84a84f583843948c01fc581c6ebcb0c71

**Comment by klhaung** [comment]
We have removed the license in commit 4ddf83d. Please help to merge the fix into SONiC. Thanks.

**Comment by lguohan** [comment]
retest this please

**Pull request merged**